src/Tribe/Admin/Manager/Service_Provider.php

```diff
@@ -151,6 +151,12 @@ public function add_attendees_view_button() {
 			return;
 		}
 
+		$supported_post_types = (array) tribe_get_option( 'ticket-enabled-post-types', [] );
```

Comment:
I think this could go directly as a parameter of `is_singular`

Reply:
Yes, exactly what I thought too, is_singular takes a string or an array of strings for post types to limit by

Reply:
Added